Source/GitReleaseManager/Attributes/SampleAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace GitReleaseManager.Core.Attributes
+{
+    using System;
+
+    [AttributeUsage(AttributeTargets.Property, AllowMultiple = false)]
+    public sealed class SampleAttribute : Attribute
+    {
+        public SampleAttribute(object value)

```

Comment:
I really like the usage of this attribute!  Not sure I would have thought to do this! Genius!